pretyping/evarsolve.mli

```diff
@@ -31,14 +40,16 @@ val expand_vars_in_term : env -> evar_map -> constr -> constr
    some problems that cannot be solved in a unique way (except if choose is
    true); fails if the instance is not valid for the given [ev] *)
 
-type conv_fun =
-  env ->  evar_map -> conv_pb -> constr -> constr -> unification_result
+type types_or_terms = bool
```

Comment:
Can you please describe what the boolean stands for and why it is relevant to the conversion (unification?) function to know that?

Reply:
I think you should also rename conv_fun into unif_fun since it is really unification.

Reply:
So the bool is `true` when `conv_fun` is  called on a term?
I'm really asking to document the API so that it is easier to understand.

May I suggest renaming `conv_fun` -> `unifier` and `conv_fun_bool` -> `conversion_check`?
And `types_or_term` -> `disable_delta` ? (or something more precise than `types_or_terms`).